terraform/eval_validate_test.go

```diff
@@ -328,7 +328,7 @@ func TestEvalValidateResource_invalidDependsOn(t *testing.T) {
 		t.Fatalf("error for supposedly-valid config: %s", err)
 	}
 
-	// No we'll make it invalid, but adding additional traversal steps
+	// Now we'll make it invalid, but adding additional traversal steps
```

Comment:
Good catch! Can you also change `but` to `by` here, while you're in the area? It should read `"Now we'll make it invalid by adding ..."`

Reply:
True, also I think one appends things "at the end", not on the end.